graders/c/Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN apt-get update \
     && groupadd sbuser && useradd -g sbuser sbuser
 
 COPY requirements.txt /requirements.txt
-RUN pip3 install --no-cache-dir --break-system-packages -r /requirements.txt
+RUN pip3 install --no-cache-dir --break-system-packages -r /requirements.txt \
+    && ln -s /usr/lib/llvm-18/lib/libclang.so.1 /usr/lib/libclang.so
```

Comment:
Can you clarify why this is needed?

Reply:
Restored and moved to previous `RUN` directive in 030b83c8773b3b3a4672c330dfe1d55850dc8852. This is needed because we don't install libclang-dev, which adds this file.